src/gui/src/layoutViewer.h

```diff
@@ -384,6 +384,10 @@ class LayoutViewer : public QWidget
   // dbu)
   qreal pixels_per_dbu_;
 
+  // holds a value in dbu added to the block bounds to ensure pin markers'
+  // text don't leave the scroll area
+  const int marker_text_offset_ = 15000;
```

Comment:
not a fan of magic numbers. how does this work if the dbuPerMicron is different?